cloudbuild.yaml

```diff
@@ -12,20 +12,23 @@ steps:
       - DOCKER_FLOW_VERSION=${_VERSION}
       - '-t'
       - ${_IMAGE_NAME}:${_IMAGE_VERSION}
+      - '-t'
+      - ${_IMAGE_NAME}:latest
```

Comment:
adding `:latest` tag to make sure any dev can use `gcr.io/taskcluster-dev/taskcluster-dev/main:latest` for their needs